pkg/server/nodemanager/node_controller.go

```diff
@@ -183,34 +184,21 @@ func (c *NodeController) doPoolsCalculation() (map[string]string, error) {
 }
 
 func (c *NodeController) getInstanceCloudInit() error {
-	cert, key, err := c.CertificateFactory.CreateNodeCertAndKey()
-	if err != nil {
-		return util.WrapError(err, "Error creating node cert")
-	}
-	certBytes, err := certs.MarshalCert(cert)
-	if err != nil {
-		return util.WrapError(err, "Error serializing node cert")
-	}
-	keyBytes, err := certs.MarshalKey(key)
-	if err != nil {
-		return util.WrapError(err, "Error serializing node key")
-	}
-	rootCertBytes, err := certs.MarshalCert(&c.CertificateFactory.Root)
-	if err != nil {
-		return util.WrapError(err, "Error serializing root cert")
-	}
-
 	c.CloudInitFile.ResetInstanceData()
-	c.CloudInitFile.AddKipFile(
-		string(rootCertBytes), path.Join(itzoDir, "ca.crt"), "0644")
-	c.CloudInitFile.AddKipFile(
-		string(certBytes), path.Join(itzoDir, "server.crt"), "0644")
-	c.CloudInitFile.AddKipFile(
-		string(keyBytes), path.Join(itzoDir, "server.key"), "0600")
-	c.CloudInitFile.AddItzoVersion(c.Config.ItzoVersion)
-	c.CloudInitFile.AddItzoURL(c.Config.ItzoURL)
-	if len(c.Config.CellConfig) > 0 {
-		c.CloudInitFile.AddCellConfig(c.Config.CellConfig)
+	if !c.Config.UseCloudParameterStore {
```

Comment:
minor / stylistic: I'd align default behavior of this function (which is `!c.Config.UseCloudParameterStore`) to right.
so instead having big block with default behavior in `if !var{}`, I'd change it to 
```
if c.Config.UseCloudParameterStore {
    return nil
}
... default behavior
```

Reply:
:+1: